#!/usr/bin/env python
from dynamic_reconfigure.parameter_generator_catkin import *
PACKAGE = "rviz_monitor"


gen = ParameterGenerator()

gen.add("route_id",    int_t,    0, "route id", 0,  0, 100)

gen.add("map_server_ip",    str_t,
        0, "map server IP",  "172.168.103.189")

gen.add("refline_color_r",    double_t,    0, "refline color red", 0,  0, 1)
gen.add("refline_color_g",    double_t,    0, "refline color green", 0,  0, 1)
gen.add("refline_color_b",    double_t,    0, "refline color blue", 1,  0, 1)

# gen.add("double_param", double_t, 0, "A double parameter",    .5, 0,   1)

# gen.add("bool_param",   bool_t,   0, "A Boolean parameter",  True)


exit(gen.generate(PACKAGE, "rviz_monitor", "myconfig"))
